inspirehep/utils/record.py

```diff
@@ -107,6 +109,15 @@ def getitem(k, v, default):
     return value
 
 
+def get_record_type(record):
+    """Get record's type e.g. 'literature', 'jobs' etc."""
+    schema = record.get('$schema')
+    if schema:
+        return InspireRecordIdProvider.schema_to_pid_type(schema)
+    else:
+        return None
```

Comment:
Fair point, `None`'s not the best option. What's the alternative though - raising a custom exception? Empty string?